crates/isograph_schema/src/validate_client_field.rs

```diff
@@ -62,6 +75,7 @@ pub(crate) fn validate_and_transform_client_fields(
 
     get_all_errors_or_all_ok_iter(client_fields.into_iter().map(|client_field| {
         match client_field {
+            ClientType::ClientPointer(_) => todo!(),
```

Comment:
maybe something like `todo!("validating client pointer selection sets is not implemented yet")` or something?

But also, in this case, would it work if we call `validate_client_field_selection_set`?